bugsnag/django/middleware.py

```diff
@@ -1,12 +1,15 @@
 from __future__ import division, print_function, absolute_import
 
+from django.utils.deprecation import MiddlewareMixin
```

Comment:
Good catch. But actually now that you point this out, passing a parameter to `object.__init__()` throws `TypeError: object.__init__() takes no parameters`, which is what will happen for Django projects < 1.10 with passing `get_response` to the parent `__init__` method.